charts/core-dump-handler/values.yaml

```diff
@@ -44,6 +44,9 @@ daemonset:
 serviceAccount:
   create: true
   name: "core-dump-admin"
+  # when running on Openshift, this creates a new SCC and attaches it to the service account
```

Comment:
worth to create another top-level entry? so instead of:

```
serviceAccount:
  createScc: true
  sccName: "core-dump-admin-privileged"
```

```
scc:
  create: true
  name: "core-dump-admin-privileged"
```

Reply:
I prefer 
```
scc:
  create: true
  name: "core-dump-admin-privileged"
```

Reply:
done, let me know